documentation/src/docs/wire-protocol.md

```diff
@@ -79,14 +79,12 @@ The `ApppendBlockEnd` has a `sizeOfWholeEvents` to allow the append block to be
 
 1.  `Data`: It contains the Event's data (only valid inside the block).
 
-
 ## Data Appended - Reply
 
 1. `writerId`: UUID (16 bytes).It identifies the requesting appender.
 2. `eventNumber`: Long (8 bytes). This matches the `lastEventNumber` in the append block.
 3. `previousEventNumber`: Long (8 bytes). This is the previous value of `eventNumber` that was returned in the last `DataAppeneded`.
 
-
```

Comment:
The current WireCommands are listed @ https://github.com/pravega/pravega/blob/master/shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java
This includes WireCommands for Transansaction, SegmentInfo and Table apis.
I assume this should be included here...

Reply:
accepted